lib/Room.ts

```diff
@@ -84,7 +84,16 @@ export const performHandover = async (modify: IModify, read: IRead, rid: string,
             throw new Error(`${Logs.HANDOVER_REQUEST_FAILED_ERROR} ${error}`);
         });
     if (!result) {
-        const offlineMessage: string = await getAppSettingValue(read, AppSetting.DialogflowServiceUnavailableMessage);
+        const offlineMessage: string = await getAppSettingValue(read, AppSetting.DialogflowServiceUnavailableMessage),
+        handoverFailure = {
+            error: offlineMessage,
+            errorMessage: 'Unable to reach Liveagent bot, it may be offline or disabled.',
+            dialogflow_SessionID: rid,
+            visitorDetails: (({ id, token }) => ({ id, token }))(visitor),
+            targetDepartment: livechatTransferData.targetDepartment
+        }
+
+        console.log('Failed to handover', handoverFailure);
```

Comment:
@ItzzRitik can you use this from now on when we add new log points please..... JSON.stringify(handoverFailure)

Reply:
Understood, will take care

Reply:
thanks....  I'll test

Reply:
@ItzzRitik I'm not yet able to get this to work....  maybe tomorrow you can demo for us how this should work.

Reply:
@ear-dev I have updated the description, with steps to reproduce, please have a look.